app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -179,7 +185,148 @@ class DefaultToolbarMenu(
+                primaryTextColor(),
+            highlight = BrowserMenuHighlight.HighPriority(
+                endImageResource = R.drawable.ic_sync_disconnected,
+                backgroundTint = context.getColorFromAttr(R.attr.syncDisconnectedBackground),
+                canPropagate = false
+            ),
+            isHighlighted = { hasAccountProblem }
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.Settings)
+        }
+
+        val desktopMode = BrowserMenuImageSwitch(
+            imageResource = R.drawable.ic_desktop,
+            label = context.getString(R.string.browser_menu_desktop_site),
+            initialState = {
+                selectedSession?.content?.desktopMode ?: false
+            }
+        ) { checked ->
+            onItemTapped.invoke(ToolbarMenu.Item.RequestDesktop(checked))
+        }
+
+        val addToTopSites = BrowserMenuImageText(
+            label = context.getString(R.string.browser_menu_add_to_top_sites),
+            imageResource = R.drawable.ic_top_sites,
+            iconTintColorResource = primaryTextColor()
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.AddToTopSites)
+        }
+
+        val addToHomescreen = BrowserMenuImageText(
+            label = context.getString(R.string.browser_menu_add_to_homescreen),
+            imageResource = R.drawable.ic_add_to_homescreen,
+            iconTintColorResource = primaryTextColor()
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.AddToHomeScreen)
+        }
+
+        val syncedTabs = BrowserMenuImageText(
+            label = context.getString(R.string.synced_tabs),
+            imageResource = R.drawable.ic_synced_tabs,
+            iconTintColorResource = primaryTextColor()
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)
+        }
+
+        val installToHomescreen = BrowserMenuHighlightableItem(
+            label = context.getString(R.string.browser_menu_install_on_homescreen),
+            startImageResource = R.drawable.ic_add_to_homescreen,
+            iconTintColorResource = primaryTextColor(),
+            highlight = BrowserMenuHighlight.LowPriority(
+                label = context.getString(R.string.browser_menu_install_on_homescreen),
+                notificationTint = getColor(context, R.color.whats_new_notification_color)
+            ),
+            isHighlighted = {
+                !context.settings().installPwaOpened
+            }
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.InstallToHomeScreen)
+        }
+
+        val findInPage = BrowserMenuImageText(
+            label = context.getString(R.string.browser_menu_find_in_page),
+            imageResource = R.drawable.mozac_ic_search,
+            iconTintColorResource = primaryTextColor()
+        ) {
+            onItemTapped.invoke(ToolbarMenu.Item.FindInPage)
+        }
+
+        val reportSiteIssuePlaceholder = WebExtensionPlaceholderMenuItem(
```

Comment:
I am wondering what happens to this in the new menu order? Was this simply forgotten or was it intentional?

Reply:
This was intentional. I can follow up with Victoria to figure out why